lib/mariaex.ex

```
@@ -291,10 +278,8 @@ defmodule Mariaex do
     case DBConnection.close(conn, query, defaults(opts)) do
       {:ok, _} ->
         :ok
-      {:error, %Mariaex.Error{}} = error ->
-        error
       {:error, err} ->
-        raise err
+        err
```

I dont think this right, don't we want to return the error tuple?